app/views/welcome/index.html.erb

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Bike Share</title>
-    <!-- <link href="https://fonts.googleapis.com/css?family=Cabin+Sketch" rel="stylesheet"> -->
+    <link href="https://fonts.googleapis.com/css?family=Tajawal" rel="stylesheet">
```

Comment:
Now I have to google what Tajawal looks like.

Reply:
Just a font placeholder until we decide on something collectively